samza-api/src/main/java/org/apache/samza/table/Table.java

```diff
@@ -38,7 +38,7 @@
  * <p>
```

Comment:
I think we should add a section to the table api page to include the partial update feature. Since this pr is already very big, I suggest open another pr for the doc change. What do you think?

Reply:
Added a ticket to track it: https://issues.apache.org/jira/browse/SAMZA-2717

Reply:
Thanks!